### DISCUSSION TITLE
《纳瓦尔宝典》书评 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#106] [Comments]
# 《纳瓦尔宝典》书评 | 夜天之书

近日在《大教堂与集市》的译者卫剑钒的推荐下阅读了这本被评价为新时代《穷查理宝典》的《纳瓦尔宝典》，其中有部分观点确实很有启发。卫 Sir 给这本《纳瓦尔宝典》专门写了两篇书评，已经把书中的精华几乎都介绍完全了。我在这里出于自己的理解和感悟，做一点微小的补充。  神书《纳瓦尔宝典：财富和幸福指南》说了些什么 我从纳瓦尔那里学到的22条

https://www.tisonkun.org/2022/06/09/the-almanack-of-naval-ravikant/

**Comment by kcfuler** [comment]
👍
